wfe2/verify.go

```diff
@@ -318,6 +316,9 @@ func (wfe *WebFrontEndImpl) parseJWS(body []byte) (*jose.JSONWebSignature, *prob
 	parsedJWS, err := jose.ParseSigned(bodyStr)
 	if err != nil {
 		wfe.stats.joseErrorCount.With(prometheus.Labels{"type": "JWSParseError"}).Inc()
+		if strings.HasPrefix(err.Error(), "failed to unmarshal JWK: square/go-jose: invalid EC private key, wrong length") {
```

Comment:
Can you add a TODO/note to this ref'ing https://github.com/square/go-jose/pull/252 so that its clear the "private key" -> "public key" switch in the `HasPrefix` check vs the returned `probs.Malformed` is intentional?

Reply:
FYI I filed an issue about this already :-D https://github.com/letsencrypt/boulder/issues/4300

Reply:
Added a TODO for this.